packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -7,7 +7,7 @@
     <button
       class={tabClass}
       onclick={select}
-      aria-label={arialLabelText}
```

Comment:
this was undefined as there was no `ariaLabelText`

Reply:
That's how unit tests fine bugs 👍